internal/controller/apiextensions/composite/reconciler.go

```diff
@@ -403,6 +404,17 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		"name", cr.GetName(),
 	)
 
+	// Check the pause annotation and return if it has the value "true"
+	// after logging, publishing an event and updating the SYNC status condition
+	if meta.IsPaused(cr) {
```

Comment:
@ulucinar I think there are a couple of other places we need to update:
* [XRM doc](https://github.com/crossplane/crossplane/blob/master/design/one-pager-managed-resource-api-design.md) should be updated to include this annotation.
* [This docs](https://github.com/crossplane/crossplane/blob/master/docs/concepts/managed-resources.md) page should document the behavior.
* Claims should also be stopped from reconciliation since this is part of XRM now.

Reply:
Thank you @muvaf. Added & tested support for pause annotation for claims also. One observation is that claims pass their annotations to XRs. I will add more observations into the description related to this behavior.

Reply:
@ulucinar I don't think the docs got an update. Could you try to add it today so that we merge it before the code freeze tomorrow?